### PR TITLE
help page

### PR DESCRIPTION
**Pull request by JojoVes**
Add Help Page

* Following guidelines set by [Drupal](https://www.drupal.org/node/632280#help)

# What does this Pull Request do?

Adds a high-level help page for users. Given that it is for users, it has no need to mention how things work behind the scenes, so I have kept it as simple as possible.

# How should this be tested?

* Review the content of the documentation and make sure links work.

### CODE REVIEW

File: xml_forms.module [hunk 1]
```diff
@@ -21,3 +23,46 @@ function xml_forms_get_schema($filename) {
...
+    case 'help.page.xml_forms':
+      // About section.
+      $output = '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('The XML Forms group of modules includes:</br>' .
+        '<ul>' .
+        '<li>Islandora XML Forms <a href=":xml_forms">wiki page*</a></li>' .
+        '<li>Islandora XML Form Builder <a href=":xml_form_builder">wiki page*</a></li>' .
+        '<li>Islandora XML Form Elements <a href=":xml_form_elements">wiki page*</a></li>' .
+        '<li>Islandora XML Form API <a href=":xml_form_api">wiki page*</a></li>' .
+        '<li>Islandora XML Schema API <a href=":xml_schema_api">wiki page*</a></li>' .
```
Comment (adam-vessey) [inline xml_forms.module]: Could use something to separate the items from the links... colons? Move to a definition list?

**Review by adam-vessey — COMMENTED** [review]
Better than what was there, though... kinda feeling like we should move to using render arrays at some point... or templates, or something? Building huge bits of markup with strings feels... less than ideal?

Shrug?

**Comment by willtp87** [comment]
As to the markup thing it doesn't really sit well with me either but it is how all the Drupal examples are written so I was giving it a pass. They must have their reasons even if it is just readability?

**Comment by JojoVes** [comment]
Yeah, I was just trying to follow what appeared to be the Drupal standards...but we could switch to a different convention if we want? That just adds the issue of consistency: if we switch conventions now, do I redo all the merged help pages?

**Comment by willtp87** [comment]
We would re-work. I can't say it feels necessary right now, it seems like a 'when in Rome' situation.

**Pull request merged**